src/main/kotlin/org/jetbrains/research/ml/coding/assistant/dataset/TaskTrackerDatasetFetcher.kt

```diff
@@ -0,0 +1,36 @@
+package org.jetbrains.research.ml.coding.assistant.dataset
+
+import com.github.doyaaaaaken.kotlincsv.dsl.csvReader
+import org.jetbrains.research.ml.coding.assistant.dataset.model.*
+import org.jetbrains.research.ml.coding.assistant.utils.getListFiles
+import java.io.File
+import kotlin.streams.toList
+
+object TaskTrackerDatasetFetcher : DatasetFetcher {
```

Comment:
Please, add a comment about requirements for the TaskTrackerDataset and some and the features of this fetcher